sys/slot_util/Makefile

```diff
@@ -0,0 +1,3 @@
+CFLAGS += -DSLOT0_OFFSET=$(SLOT0_OFFSET)
```

Comment:
I think this file and folder needs to be deleted, right?

Reply:
yup, done, thanks!